### PR TITLE
Import libbf for BigInt

### PR DESCRIPTION
**Pull request by ksh8281**

### CODE REVIEW

File: build/escargot.cmake [hunk 1]
```diff
@@ -89,6 +89,32 @@ ADD_SUBDIRECTORY (third_party/GCutil)
...
+TARGET_INCLUDE_DIRECTORIES (libbf PUBLIC ${ESCARGOT_THIRD_PARTY_ROOT}/libbf)
+SET (LIBBF_CFLAGS
+	${ESCARGOT_GCUTIL_CFLAGS} # we can share arch flags with gcutil
+	${CFLAGS_FROM_ENV}
+	-w
+	-g3
+    -fdata-sections
+    -ffunction-sections
+    -fno-omit-frame-pointer
+    -fvisibility=hidden)
```
Comment (clover2123) [inline build/escargot.cmake:106]: Just a minor correction.
Would you please keep the indentation at here?

Comment (clover2123) [reply build/escargot.cmake:106]: One more thing,
It seems that only a few c files are used by Escargot. So how about excluding any other unnecessary files like `sha1sum` files (maybe checksum files?)

**Review by clover2123 — CHANGES_REQUESTED** [review]

File: build/escargot.cmake (hunk 1 above)
Comment (ksh8281) [reply build/escargot.cmake:106]: Sounds great. I am going to remove unused files

**Comment by ksh8281** [comment]
I updated PR

**Review by bbrto21 — APPROVED** [review]
LGTM

**Review by clover2123 — APPROVED** [review]
LGTM

**Pull request merged**